Make.inc

```diff
@@ -132,6 +132,10 @@ else
 JULIA_COMMIT := $(JULIA_VERSION)
 endif
 
+# Increase these every release when breaking ABI in a backward-incompatible way
```

Comment:
I don't think you can make the major version a decimal like that, so I'm opting instead to set the major version number to `Y` in Julia's `X.Y.Z` version number.  Once we hit 1.0, we'll add `W` to the major soversion, where `W` is the number of major releases before 1.0, so that we continue to count up with major releases.